src/android/aidl/example-service/service-bindings.md

```diff
@@ -9,7 +9,7 @@ _birthday_service/aidl/com/example/birthdayservice/IBirthdayService.aidl_:
 }
 ```
 
-_out/soong/.intermediates/.../birthdayservice/IBirthdayService.rs_:
+_out/soong/.intermediates/comprehensive-rust/.../com_example_birthdayservice.rs_:
```

Comment:
Can we do this to hint that "you find `com_example_birthdayservice.rs` somewhere in the `.intermediates` directory"?

```suggestion
_out/soong/.intermediates/.../com_example_birthdayservice.rs_:
```